src/main/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClient.java

```diff
@@ -50,6 +50,7 @@ public PDFServiceClient(URI pdfServiceBaseUrl, String version) {
                 requestEntityFor(template, placeholders),
                 byte[].class);
         } catch (HttpClientErrorException e) {
+            LOGGER.warn(e.getMessage(), e);
```

Comment:
it's up to the client how they handle this, it's already being thrown up they can then choose to log it or not IMO.

Reply:
thanks, removing it